pkg/clients/clients.go

```diff
@@ -172,6 +172,7 @@ func setupFeatureGatesAccessor(
 			Namespace:  operatorNamespace,
 			Name:       "cluster-olm-operator",
 		},
+		nil,
```

Comment:
Still have reservations about supplying a nil PassiveClock here.

Reply:
https://github.com/openshift/cluster-olm-operator/blob/4a5b8c1578f4befccdef4cd21bd99a480a1400f8/pkg/clients/clients.go#L109C1-L110C1 establishes a clock which can be passed into this method by widening the signature and passing in `opClient.Clock` to this function.

Reply:
Oops yea I missed this, thanks for catching it. I just passed in `clock.RealClock{}` since we're using this no-op clock anyway, and that doesn't feel like it's enough justification to widen the function signature.

Reply:
I mean ... it's not an exported interface.

Reply:
agreed, but it still feels like "effort for nothing", no?

Reply:
I'd almost always prefer to perpetuate an object instance rather than get in the habit of needlessly generating new instances.  
So... no.